tutorials/experiments/prompt-optimization.ipynb

```diff
@@ -0,0 +1,2121 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    base_template = """
</pre><p>We can use Prompts now!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/6472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>